sim/src/main/cc/bridges/BridgeHarness.h

```diff
@@ -0,0 +1,43 @@
+#ifndef MIDAEXAMPLES_BRIDGEHARNESS_H
+#define MIDAEXAMPLES_BRIDGEHARNESS_H
+
+#include "simif.h"
+#include "simif_peek_poke.h"
+
+class bridge_driver_t;
```

Comment:
For my own understanding, why the declaration here instead of importing from bridge_driver?

Reply:
In general, it is a good idea to use forward declarations in headers to avoid unnecessary dependencies. If some C++ file including `BridgeHarness.h` does not actually need to know about the implementation of the bridge driver, it can be compiled without unnecessarily including the header.